awslabs/aws-c-http
# More local host test

- Add a non-tls h2 server script and test our client with prior knowledge against it
- Stream Manager stress test against local host
- Test download and upload
  - download of 2.5G file works well
     - 85.95 sec for linux 
     - 58.53 sec for mac 
     - 62.65 sec for windows
  - Upload of 2.5G file is weirdly slow for linux.
  - Now using 0.25G upload
    - 162.37 sec for linux 
    - 6.43 sec for mac
    - 5.44 sec for windows
  - For upload performance, we tested against nginx server for mac and linux. We didn't see any performance issue around them. So, we assume the performance issue is related to hyper h2 implementation.
  - Upload 0.25G data to nginx and seems like nginx will store the data in disk as temp file.
    - 9.02 sec for mac
    - 4.00 sec for linux

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
